netbsd/Platform.c

```diff
@@ -0,0 +1,324 @@
+/*
+htop - netbsd/Platform.c
+(C) 2014 Hisham H. Muhammad
+(C) 2021 Santhosh Raju
+Released under the GNU GPLv2, see the COPYING file
+in the source distribution for its full text.
+*/
+
+#include "Platform.h"
```

Comment:
Include path relative to main directory.

Reply:
Should be fixed in d52c724